packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -675,6 +675,9 @@ abstract class FlutterDriver {
   /// With frame sync disabled, it's the responsibility of the test author to
   /// ensure that no action is performed while the app is undergoing a
   /// transition to avoid flakiness.
+  /// See also:
+  ///
+  /// * [FlutterDriver.waitFor], which waits until target is located.
```

Comment:
Based on this comment its unclear to me why I should look at waitFor and how it relates to runUnsynchronized,

Reply:
@goderbauer Added this per the original issue [description](https://github.com/flutter/flutter/issues/51971#issue-575853935).

Reply:
The issue is asking for the link in the opposite direction: `waitFor` should link to `runUnsynchronized` and needs to describe how the two concepts relate.